styleguide/components/41-organisms/feedback-form/feedback-form.twig

```diff
@@ -0,0 +1,39 @@
+<div class="feedback-form {{ modifier }}">
+  <h3><button>{{ 'Can we improve something about this page?' }}<i class="icon-chevron-down" aria-hidden="true"></i></button></h3>
+  <div class="content" aria-hidden="true">
+    <div class="info-box">
+      {% include '@image' with {
+        'src': '/styleguide/img/svg/gentinfo.svg',
+        'alt_text': 'Gentinfo icon',
+        'modifier': 'info-box-icon'
+      } %}
+      <div class="info-box-text">
+        {% include '@h4' with {
+          'h4_text': 'You can contact Gentinfo for personal questions or complaints.'
+        } %}
+
+        {% include '@paragraph' with {
+          'text': 'You can contact Gentinfo from Monday to Saturday from 08:00 till 19:00<br />
+            09 210 10 10<br />
+            <a href="mailto:Gentinfo@stad.gent">Gentinfo@stad.gent</a>'
```

Comment:
This looks crooked because of the link padding.
Suggest reusing the fix you created for the topic teasers, it's clear we'll need this in a lot of places.

Reply:
This goes back to design next week... this is not a good solution imo.